### PR TITLE
Picture tag Support

### PR DESCRIPTION
**Pull request by FarhanShoukat**

### CODE REVIEW

**Comment by FarhanShoukat** [comment]
[Image optimization 2.0](https://trello.com/c/ytve55Hk/924-image-optimization-20)

File: delta/html.py [hunk 1]
```diff
@@ -258,6 +258,74 @@ def classes_check(op):
...
+        _create_img_tag(noscript_tag, 'src')
+    else:
+        _create_img_tag(picture_tag, 'src')
+
+    return figure_tag
+
+
+@format
+def picture(root, op):
+    return base_picture(root, op)
```
Comment (stevenmoseley) [inline delta/html.py]: `base_picture()` isn't reall necessary if it's only called here. Why not just have all the code in `picture()`?

**Review by stevenmoseley — CHANGES_REQUESTED** [review]

File: delta/html.py [hunk 2]
```diff
@@ -258,6 +258,74 @@ def classes_check(op):
...
+    figure_tag = sub_element(root, 'figure')
+
+    a = None
+    href = attributes.get('link')
+    if href:
+        a = sub_element(figure_tag, 'a')
+        a.attrib['href'] = href.get('url')
+        if href.get('openNewTab'):
+            a.attrib['target'] = '_blank'
+            a.attrib['rel'] = 'noopener noreferrer'
```
Comment (stevenmoseley) [inline delta/html.py]: Isn't a lot of this code reused from the `base_image` function? Is there a simple way to have a shared set of functions that they can both use?

**Review by stevenmoseley — CHANGES_REQUESTED** [review]

**Comment by FarhanShoukat** [comment]
@stevenmoseley made requested changes.

**Review by stevenmoseley — APPROVED** [review]

**Pull request merged**